images/demo/jupyter/.kamuconfig

```diff
@@ -4,7 +4,6 @@ kind: CLIConfig
 content:
   engine:
     runtime: podman
-    networkNs: host
```

Comment:
I think the necessity for using host networking came from kubernetes. As Jupyter runs inside k8s pod it seemed not to be able to create another network NS inside it.

I don't see testing under k8s in your test plan, so I'd recommend:
- Extracting the full podman run command and testing it in `jupyter.demo.kamu.dev` environment
- You can run spark container manually and then see if our python client can [connect to it](https://github.com/kamu-data/kamu-client-python/tree/master?tab=readme-ov-file#using-with-spark)